thirdparty/s3.go

```diff
@@ -11,20 +11,22 @@ import (
 )
 
 const (
-	//PresignExpireTime sets the amount of time the link is live before expiring
+	// PresignExpireTime sets the amount of time the link is live before expiring.
 	PresignExpireTime = 24 * time.Hour
-	NotFoundError     = "NotFound"
+	// NotFound is returned by s3 when headObject doesn't exist
```

Comment:
Period at the end of the sentence.

I would phrase this more generally as "returned by S3 when an object does not exist" rather than assuming it specifically only applies to the `GetHeadObject` operation.

Reply:
Bump the first comment about the period.